awxbench/awxbench.yml

```diff
@@ -0,0 +1,82 @@
+- hosts: database
+  environment:
+    PATH: "/opt/rh/rh-postgresql10/root/usr/bin/:{{ ansible_env.PATH }}"
+    LD_LIBRARY_PATH: "/opt/rh/rh-postgresql10/root/usr/lib64"
+  tasks:
+    - name: Disable SELinux
+      selinux:
+        state: disabled
```

Comment:
Yikes. Why is this needed?

Reply:
I need to figure out the proper `selinux` "things" to set on the `.csv` file that I upload so that postgres can access it via the `COPY FROM` statement. I was reading the selinux coloring book. I think I know how, just ran out of time.

Reply:
Any chance running `restorecon` on those files fixes it?